indico/modules/events/abstracts/forms.py

```diff
@@ -498,7 +498,8 @@ def __init__(self, *args, **kwargs):
         if not is_invited:
             self.person_links.require_speaker_author = abstracts_settings.get(self.event, 'speakers_required')
             self.person_links.allow_speakers = abstracts_settings.get(self.event, 'allow_speakers')
-            self.person_links.disable_user_search = session.user is None
+            self.person_links.disable_user_search = (not config.ALLOW_PUBLIC_USER_SEARCH and
+                                                     not self.event.can_manage(session.user))
```

Comment:
This is missing the `or session.user is None` - we never allow unauthenticated users to search.

(even though I'm not sure whether they can even get to the point where they get to the search dialog without being asked to login first...)

Reply:
It's required to be logged in but this is future-proof, in case it's ever allowed to submit abstracts without being logged.